### PR TITLE
CheesyLingo for FFaker

### PR DESCRIPTION
**Pull request by F3PiX**
Provides CheesyLingo, with 2 word titles (eg for fake Team names), words, short 'cheesy' phrases and longer paragraphs. Cheesy - but never tacky.

### CODE REVIEW

File: lib/ffaker/cheesy_lingo.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# encoding: utf-8
+
+module FFaker
+  module CheesyLingo
+
+    # Born during RailsGirls Summer of Code 2015. Team Cheesy says Hi!
```
Comment (andremedeiros) [inline lib/ffaker/cheesy_lingo.rb]: :tada: :balloon: :cake:

**Comment by andremedeiros** [comment]
:+1:

**Comment by rocknruby** [comment]
Sweet. Maybe it helps someone :+1:

**Comment by F3PiX** [comment]
:tada: Thanks!!

**Pull request merged**